sale_product_multi_add/wizard/sale_import_products.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# © 2016 Cédric Pigeon, ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
+import openerp.addons.decimal_precision as dp
+
+UNIT = dp.get_precision('Product Unit of Measure')
```

Comment:
I would not put this in a global constant, because the result of get_precision is actually a method that can't really be used outside of a field declaration.